### ISSUE TITLE
Symbols are prefixed with ?? when it seems they're otherwise fine

### ISSUE THREAD

**Issue by jasonmalinowski** [microsoft/perfview#162]
I'm looking at some dumps traces symbol names are prefixed with ?? when the symbols appear totally fine. Is there a way to turn this flagging off? I can use `{%}!??{*}->$1!$2` as a group pattern to work around it but unfortunately it gets messy if I'm trying to do other group patterns as well.

**Comment by vancem** [comment]
The ?? prefix is indicating that the symbol is failing some basic validity checks.   

See FindNameForRva in SymbolReader.cs for the code that adds this.   

Basically we have an address and we ask the PDB reader to find the symbol, and it gives us something, but that symbol also has a start and end extent, and the returned symbol is NOT in that range.   Thus something is fishy.   

We added this extra check because we got complaints that PerfView was showing BOGUS symbols.  That is VERY BAD.   However when we filter to only the 'GOOD' ones that pass the check above, we lose many symbols that at least seem to be OK.    The patch was to prefix the non-GOOD ones with ?? so you have some indicator that it is suspect.  

This mostly requires someone with experience with the MSPDB apis to figure out what is going on.   It is worth doing.
